src/modules/utils/panel/panels/rrdglcd/RrdGlcd.cpp

```diff
@@ -297,6 +316,68 @@ void RrdGlcd::renderGlyph(int xp, int yp, const uint8_t *g, int pixelWidth, int
     }
 }
 
+void RrdGlcd::pixel(int x, int y, int color)
+{
+    if (y < HEIGHT && x < WIDTH)
+  {
+    unsigned char mask = 0x80 >> (x % 8);
+    unsigned char *byte = &fb[y * (WIDTH/8) + (x/8)];
+    if ( color == 0 )
+        *byte &= ~mask; // clear pixel
+    else
+        *byte |= mask; // set pixel
+	    // Change the dirty rectangle to account for a pixel being dirty (we assume it was changed)
+    if (startRow > y) { startRow = y; }
```

Comment:
These next 4 variables appear to never be used for anything, so why waste RAM by defining them?

Reply:
Removed unused variables.